HandheldCompanion/Utils/WindowSinkerUtils.cs

```diff
@@ -0,0 +1,144 @@
+﻿using System;
```

Comment:
Why do we have yet more sinker-like code? There already is code to get HC on top.

Reply:
I made this before finding the HC sinker code. My Ally was crashing when trying to open Quick Menu and thought it could be related

Reply:
Best to remove it then